pkgs/development/python-modules/wxPython/4.1.nix

```diff
@@ -0,0 +1,124 @@
+{ lib
+, stdenv
+, openglSupport ? true
+
+, fetchPypi
+, buildPythonPackage
+, which
+, pkgconfig
+, gtk3
+, pkg-config
+
+, python
+, isPy27
+, freeglut
+, gst_all_1
+, libpng
+, libtiff
+, libjpeg
+, libnotify
+, SDL2
+, xorg
+
+, libX11
+, pyopengl
+, requests
+, doxygen
+, cairo
+, ncurses
+, pango
+, pathlib2
+
+, libGLU
+}:
+let
+  dynamic-linker = stdenv.cc.bintools.dynamicLinker;
+in
+buildPythonPackage rec {
+  pname = "wxPython";
+  version = "4.1.0";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12x4ci5q7qni4rkfiq6lnpn1fk8b0sfc6dck5wyxkj2sfp5pa91f";
+  };
+
+  # https://github.com/NixOS/nixpkgs/issues/75759
+  # https://github.com/wxWidgets/Phoenix/issues/1316
+  doCheck = false;
+
+  nativeBuildInputs = [ pkgconfig which doxygen gtk3 pkg-config ];
+
```

Comment:
pkgconfig and pkg-config are aliases of the same package, please prefer `pkg-config`
```suggestion
  nativeBuildInputs = [ which doxygen gtk3 pkg-config ];

```